tests/src/core/users/DetermineLifetimeConditions.test.ts

```diff
@@ -83,10 +19,10 @@ describe('Determining Lifetime conditions', () => {
       const tierNotFoundError = new TierNotFoundError('Old product was found');
       const mockedUser = getUser();
 
-      jest.spyOn(paymentService, 'getUserSubscription').mockResolvedValue({ type: 'free' });
-      jest.spyOn(tiersService, 'getTierProductsByProductsId').mockRejectedValue(tierNotFoundError);
+      jest.spyOn(services.paymentService, 'getUserSubscription').mockResolvedValue({ type: 'free' });
```

Comment:
Consider destructuring the `services` object, reducing the LOCs changed and making it easier/shorter to read